### DISCUSSION TITLE
HOW TO ADD CAMERA FOLLOWS CHARACTER ON NOTE HIT EFFECT ON KADE ENGINE

### DISCUSSION THREAD

**Discussion by Mlaofmd** [KadeArchive/Kade-Engine#2894] [Q&A] [answered]
Hello, I making the mod ask me please how to add camera follows character on note hit effect on kade engine 1.8 look like vs sonic.exe

**Comment by RealDealJinx** [comment, marked as answer]
https://gamebanana.com/tuts/14591 ??
